src/commands/fun/randommeme.js

```diff
@@ -0,0 +1,29 @@
+const SA = require('superagent')
+const Command = require('../../classes/Command')
+
+module.exports = new Command(async msg => {
+  try {
+    const res = await SA.get(`https://api.imgur.com/3/g/memes/viral/${Math.floor((Math.random() * 8) + 1)}`)
+      .set('Authorization', 'Client-ID ' + process.env.IMGUR_KEY)
+    const ctx = res.body.data[Math.floor((Math.random() * 20) + 1)]
+    await msg.channel.createMessage({
+      embed: {
+        url: ctx.link,
+        title: ctx.title,
+        image: {
+          url: ctx.images[0].link
+        },
+        footer: {
+          text: 'imgur.com',
+          icon_url: 'https://i.imgur.com/JFa7xb3.png'
+          // imgur doesnt have easily accessible versions of their logo
+          // improvide, adapt, overcome
+        },
+        color: 0x1bb76e
```

Comment:
It might be worth investigating a hex to integer color function to improve clarity with embed colors.

Reply:
In my opinion hex notation is more clear than ints, given that hex is one of the standard ways to convey color.

Reply:
I agree with @Dougley, hex notations are a lot clearer than ints. However, if we wanted, we could define these embed colors as web colors (`#rrggbb`) and then convert  those to hex, since then there isn't something like the `0x` messing with your head. That's not strictly necessary in my opinion however, it's perfectly adequate as is so long as the format is consistent.

Reply:
That seems entirely stylistic, and ultimately redundant. I'd much prefer keeping this as-is.